code/style.css

```diff
@@ -1,4 +1,385 @@
 html {
-  background: #0025ff;
+  background: white;
   color: #fff;
+  font-family: 'Roboto Mono', monospace;
 }
+
+.wrapper{
+  justify-content: center;
+  display: flex;
+flex-direction: column;
+animation: fadeIn 5s;
+opacity: 1;
+}
+@keyframes fadeIn{
+  from{
+    opacity: 0;
+  }
+  to{
+    opacity: 1;
+  }
+  }
+
+header{
+display: flex;
+justify-content: space-evenly;
+align-self: center;
+align-items: center;
+flex-direction: row;
+}
+
+.meny{
+  display: inline;
+}
+.menu-line{
+  width: 35px;
+  height: 5px;
+  background-color: #25838A;
+  margin: 6px 0;
+  border-radius: 10px;
+  align-self: flex-end;
+}
+.nav1:hover, .nav2:hover, .nav3:hover, .nav4:hover{
+text-decoration:underline;
+color: #25838A;
+}
+.logo{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.logo-big{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.big-news{
+  width: 100%;
+  /* border: 2px solid #25838A; */
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  max-width: 500px;
```

Comment:
is all this code for the animation of the logo? is it for all different browsers or why is it so long? the result looks professional though!